ci/build-win32.ps1

```diff
@@ -6,11 +6,74 @@ if (-not (Test-Path $subprojects)) {
+$url = "https://storage.googleapis.com/shaderc/badges/build_link_windows_vs2019_release.html"
+$shaderc = "shaderc.zip"
+$resp = Invoke-WebRequest -Uri $url
+if ($resp.Content -match '<meta http-equiv="refresh" content="\d+; url=(?<url>[^"]+)"') {
+    $url = $matches['url']
+}
+Invoke-WebRequest -Uri $url -OutFile $shaderc
+if (Test-Path "$subprojects/shaderc") {
+    Remove-Item -LiteralPath "$subprojects/shaderc" -Force -Recurse
+}
+Expand-Archive -Path $shaderc -DestinationPath "$subprojects/shaderc"
+Move-Item -Path "$subprojects/shaderc/install/*" -Destination "$subprojects/shaderc"
+
+Set-Content -Path "$subprojects/shaderc/meson.build" -Value @"
+project('shaderc', 'c', version: '2024.1')
+cc = meson.get_compiler('c')
+shaderc_dep = declare_dependency(
+  dependencies: cc.find_library('shaderc_combined', dirs: meson.current_source_dir() / 'lib'),
+  include_directories: include_directories('include')
+)
+meson.override_dependency('shaderc', shaderc_dep)
+"@
+
+# Manually wrap spirv-cross for CMAKE_MSVC_RUNTIME_LIBRARY option
+# This also allows us to link statically
+if (-not (Test-Path "$subprojects/spirv-cross-c-shared")) {
+    New-Item -Path "$subprojects/spirv-cross-c-shared" -ItemType Directory | Out-Null
+}
+Set-Content -Path "$subprojects/spirv-cross-c-shared/meson.build" -Value @"
+project('spirv-cross', 'cpp', version: '0.59.0')
+cmake = import('cmake')
+opts = cmake.subproject_options()
+opts.add_cmake_defines({
+    'CMAKE_MSVC_RUNTIME_LIBRARY': 'MultiThreaded',
+    'CMAKE_POLICY_DEFAULT_CMP0091': 'NEW',
+    'SPIRV_CROSS_EXCEPTIONS_TO_ASSERTIONS': 'ON',
+    'SPIRV_CROSS_CLI': 'OFF',
+    'SPIRV_CROSS_ENABLE_TESTS': 'OFF',
+    'SPIRV_CROSS_ENABLE_MSL': 'OFF',
+    'SPIRV_CROSS_ENABLE_CPP': 'OFF',
+    'SPIRV_CROSS_ENABLE_REFLECT': 'OFF',
+    'SPIRV_CROSS_ENABLE_UTIL': 'OFF',
+})
+spirv_cross_proj = cmake.subproject('spirv-cross', options: opts)
+spirv_cross_c_dep = declare_dependency(dependencies: [
+    spirv_cross_proj.dependency('spirv-cross-c'),
+    spirv_cross_proj.dependency('spirv-cross-core'),
+    spirv_cross_proj.dependency('spirv-cross-glsl'),
+    spirv_cross_proj.dependency('spirv-cross-hlsl'),
+])
+meson.override_dependency('spirv-cross-c-shared', spirv_cross_c_dep)
+"@
+
+if (-not (Test-Path "$subprojects/packagefiles/luajit")) {
+    New-Item -Path "$subprojects/packagefiles/luajit" -ItemType Directory | Out-Null
+}
+git clone https://github.com/benoit-pierre/wrapdb --depth 1 -b pr/add_luajit wrapdb_luajit
+Copy-Item -Path "wrapdb_luajit/subprojects/luajit.wrap" -Destination "$subprojects"
+Copy-Item -Path "wrapdb_luajit/subprojects/packagefiles/luajit/*" `
+-Destination "$subprojects/packagefiles/luajit" -Recurse -Force
+
 $projects = @(
     @{
         Path = "$subprojects/ffmpeg.wrap"
-        URL = "https://gitlab.freedesktop.org/gstreamer/meson-ports/ffmpeg.git"
-        Revision = "meson-6.1"
+        URL = "https://gitlab.freedesktop.org/kasper93/ffmpeg.git"
```

Comment:
thought: depending on how permanent this is we might want to move this into a repo inside the mpv org

Reply:
I hope it is only temporary, I've sent patches upstream, I don't intend to maintain this in a fork.